### ISSUE TITLE
PHP8 problem with each() function

### ISSUE THREAD

**Issue by S0FTWEX** [OpenMage/magento-lts#1553]
### Preconditions (*)

1. OpenMage 20.0.8
2. PHP 8.0.3

### Steps to reproduce (*)

1. In Admin Panel, go to System -> Configuration ->Advanced -> Admin
2. Page goes white blank with this error in log:
```
PHP Fatal error:  Uncaught Error: Call to undefined function each() in /app/code/core/Mage/Adminhtml/Model/System/Config/Source/Admin/Page.php:127
Stack trace:
#0 /app/code/core/Mage/Adminhtml/Model/System/Config/Source/Admin/Page.php(118): Mage_Adminhtml_Model_System_Config_Source_Admin_Page->_buildMenuArray()
#1 /app/code/core/Mage/Adminhtml/Model/System/Config/Source/Admin/Page.php(118): Mage_Adminhtml_Model_System_Config_Source_Admin_Page->_buildMenuArray()
#2 /app/code/core/Mage/Adminhtml/Model/System/Config/Source/Admin/Page.php(42): Mage_Adminhtml_Model_System_Config_Source_Admin_Page->_buildMenuArray()
#3 /app/code/core/Mage/Adminhtml/Block/System/Config/Form.php(463): Mage_Adminhtml_Model_System_Config_Source_Admin_Page->toOptionArray()
#4 /app/code/core/Mage/Adminhtml/Block/System/Config/Form.php(229): Mage_Adminhtml_Block_System_Config_Form->initFields()
```

### Solution

It would be great to finally merge pull request #1391 (or at least the files from this PR with each() functions inside).

**Comment by fballiano** [comment]
https://github.com/OpenMage/magento-lts/pull/1391 has been merged and released in 20.0.10 so i think we can close this issue

**Comment by S0FTWEX** [comment]
I agree, issue can be closed, problem has been fixed after merging #1391.
